### PR TITLE
Added Customer Dealer Crud

### PR DESCRIPTION
**Pull request by MuhammadHaseebOdho**

### CODE REVIEW

File: src/constant/CustomerDealerEnum.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * and open the template in the editor.
+ */
+package constant;
+
+/**
+ *
+ * @author Haseeb
+ */
+public enum CustomerDealerEnum {
+    Select_TYPE, Dealer, Customer;
```
Comment (govindkmk) [inline src/constant/CustomerDealerEnum.java]: May be we don't write Select_Type here.
and make it in Capital words.

File: src/model/CustomerDealer.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    private String customerDealerName;
+    private CustomerDealerEnum customerDealerEnum;
+    private String email;
+    private String phoneNumber;
+    private String address;
+    private Date createDate;
+    private Date modifiedDate;
+    private String createdBy;
+    private String updatedBy;
+    private int active;
```
Comment (govindkmk) [inline src/model/CustomerDealer.java]: make active as boolean.

File: src/constant/CustomerDealerEnum.java (hunk 1 above)
Comment (RaheemBux) [reply src/constant/CustomerDealerEnum.java]: aggred

File: src/model/CustomerDealer.java (hunk 2 above)
Comment (RaheemBux) [reply src/model/CustomerDealer.java]: aggred and also connect with @TouqeerAli  for auditable class

**Review by govindkmk — APPROVED** [review]

**Review by RaheemBux — APPROVED** [review]

**Pull request merged**